test/e2e/hydra-for-macOS.yaml

```diff
@@ -0,0 +1,5 @@
+strategies:
+  access_token: jwt
+urls:
+  self:
+    issuer: http://docker.for.mac.localhost:4444/
```

Comment:
This will only work for people using Docker for Mac, which depends now on paid licensing for big enough companies.